docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -1,13 +1,17 @@
 == Installing Jenkins plugins
-NOTE: Plugins for Jenkins can be found at https://plugins.jenkins.io
+Digger installs required Jenkins plugins by default. If you need to install more Jenkins plugins, they can be installed using Jenkins UI (recommended) or by modifying variables in certain Ansible file. See instructions below.
 
-Digger will by default install required Jenkins plugins. If more Jenkins plugins are required, they can be installed by listing them in a task in Ansible playbook or using Jenkins UI.
+NOTE: Jenkins Plugins homepage: https://plugins.jenkins.io
 
-=== Using Ansible Playbook
-https://github.com/aerogear/aerogear-digger-installer/blob/master/configure-buildfarm/defaults/main.yml[Digger Installer Ansible playbook^]. 
-The `jenkins_plugins` play can be modified to include new plugins. 
+=== Using Jenkins UI (Recommended)
+Go to `Manage Jenkins -> Manage Plugins`. If the plugin is not available in the
```

Comment:
@pmacko1 not sure if it's appropriate to document this. Is there somewhere in the Jenkins documentation that we could point to instead?